js/app.js

```diff
@@ -30,6 +30,18 @@ phpEngine.configure({
 // Expose jQuery to PHPland
 phpEngine.expose($, 'jQuery');
 
+// Expose Window to PHPland
+var this_window = window ;
```

Comment:
All four exposes in one short block of code, with one comment for all four, and using all inline variables.